lib/game.rb

```diff
@@ -1,54 +1,37 @@
 # frozen_string_literal: true
 
+require 'paint'
+require_relative './player.rb'
+require_relative './board.rb'
+
 class Game
-  attr_reader :winner
-  def initialize
-    @board = [1, 2, 3,
-              4, 5, 6,
-              7, 8, 9]
-    @winner = ''
+  def initialize(player1, player2)
+    @player1 = Player.new(player1, 'X')
+    @player2 = Player.new(player2, 'O')
+
+    @board = Board.new(@player1, @player2)
   end
 
-  def winner?
-    return true if check_array?('rows')
-    return true if check_array?('columns')
+  def play
+    loop do
+      @board.print
 
-    diagonals = [
-      [@board[0], @board[4], @board[8]],
-      [@board[2], @board[4], @board[6]]
-    ]
-    diagonals.each do |diagonal|
-      return true if check_consecutive?(diagonal)
-    end
+      @player1.move(@board.board)
+      break if @board.winner? || @board.tie?
 
-    false
-  end
+      @board.print
 
-  private
-
-  def check_array?(orientation)
-    3.times do |row|
-      array = []
-      3.times do |col|
-        array << if orientation == 'columns'
-                   @board[col * 3 + row]
-                 else
-                   @board[row * 3 + col]
-                 end
-      end
-      return true if check_consecutive?(array)
+      @player2.move(@board.board)
+      break if @board.winner? || @board.tie?
     end
-    false
-  end
 
-  def check_consecutive?(orientation)
-    if orientation.all? { |cell| cell == 'X' }
-      @winner = 'X'
-      return true
-    elsif orientation.all? { |cell| cell == 'O' }
-      @winner = 'O'
-      return true
+    if @board.winner?
```

Comment:
This logic should be inside of a method

Reply:
HI @Ispirett, thanks for your code review, however we didn´t understand quite well this last point from the review. Here you have our [game file](https://github.com/nicolasmarino99/Tic-tac-toe-Ruby/blob/board/lib/game.rb). Thanks again for you review. Have a great day!